src/chapters/booleans-and-conditionals/booleans.rst

```diff
@@ -0,0 +1,209 @@
+
+   Consider a banking application that can remind you when a bill is due. The application will notify the user that a bill is due soon, but *only if* the bill has not already been paid.
+
+In this example, we can phrase the condition as follows: Send a notification of an upcoming bill if the statement "the bill is unpaid" is true. In order to formalize such a statement in code, we need a notion of what it means to be true or false.
+
+Boolean Values
+--------------
+
+.. index:: ! true, ! false, ! boolean
+
+.. index::
+   single: boolean; value
+
+The JavaScript data type for storing true and false values is ``boolean``, named after the British mathematician, George Boole. George Boole created `Boolean Algebra <https://en.wikipedia.org/wiki/Boolean_algebra>`_ which is the basis of all modern computer arithmetic.
+
+There are only two **boolean values**. They are ``true`` and ``false``. As always, JavaScript is case-sensitive, so ``True`` and ``False`` are not valid boolean values.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(true);
+      console.log(typeof true);
+      console.log(typeof false);
+
+   **Output**
+
+   ::
+
+      true
+      boolean
+      boolean
+
+It is extremely important to realize that ``true`` and ``false`` are **not** strings. Surrounding the words "true" and "false" in quotes in JavaScript will result in string values rather than boolean values. 
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(typeof true);
+      console.log(typeof "true");
+
+   **Output**
+
+   ::
+
+      boolean
+      string
+
+Boolean Conversion
+------------------
+
+.. index:: ! Boolean(), type conversion
+
+As with the number and string data types, the boolean type has a type conversion function, ``Boolean()``. It works similarly to ``Number()`` and ``String()``, attempting to convert a non-boolean value to a boolean.
+
+.. admonition:: Try It!
+
+   Explore how ``Boolean()`` converts various non-boolean values.
+
+   .. sourcecode:: js
+
+      console.log(Boolean("true"));
+      console.log(Boolean("TRUE"));
+      console.log(Boolean(0));
+      console.log(Boolean(1));
+      console.log(Boolean(''));
+      console.log(Boolean('LaunchCode'));
+
+   `Run this program at repl.it <https://repl.it/@launchcode/Boolean-Type-Conversion>`_
+
+   Under which conditions does ``Boolean()`` convert a string to ``true``?
+
+   #. Only when the string is ``"true"``.
+   #. Whenever the string contains any non-whitespace character.
+   #. Whenever the string in non-empty.
+   #. Never. It converts all strings to ``false``.
+
+
+Boolean Expressions
+-------------------
+
+.. index::
+   single: boolean; expression
+
+.. index::
+   single: operator; equality
+
+.. index:: ! ==
+
+A **boolean expression** is an expression that evaluates to a boolean value. The equality operator, ``==``, compares two values and produces a boolean value related to whether the two values are equal to one another.
```

Comment:
I'll defer to Sally's comment for the first sentence.  ;-)

For the second sentence, how about, "The equality operator, ``==``, compares two values and returns ``true`` or ``false`` depending on whether the values are equal."

Reply:
The difference between "related to" and "depending on" is subtle, but intentional. In JS, `==` doesn't really align with the general concept of equality, so I chose to say "related to." We introduce the difference between `==` and `===` later on, but at this point I don't want to make a statement that could be misleading, even if students are unlikely to pick up on it. Thoughts on other ways to rephrase to make the statement less awkward?

Reply:
Since ``==`` vs. ``===`` is discussed later, how about keeping the concrete "returns ``true`` or ``false`` depending on...", but using ``>`` or ``<`` instead of ``==``?

Reply:
I have come around to your original suggestion on this. There are a few other places on this page where we talk about `==` and it would be very awkward to say "depending on..." in each case. Additionally, there is a warning admonition that hints at the underlying reality.